src/components/SummaryList.tsx

```diff
@@ -4,12 +4,15 @@ interface SummaryListProps {
   children: ReactNode;
 }
 
+interface SummaryListItemAction {
+  text: string;
+  href: string;
+}
+
 interface SummaryListItemProps {
   labelText: string;
   children: ReactNode;
-  href?: string;
-  actionText?: string;
-  actionValue?: string;
+  actions?: SummaryListItemAction[];
```

Comment:
Allows `SummaryListItem` to be extended with different actions by the caller, e.g. `Change` and `Delete` etc.